package.json

```diff
@@ -364,8 +364,8 @@
 				"category": "OpenShift"
 			},
 			{
-				"command": "openshift.explorer.switchContext",
-				"title": "Switch Contexts",
+				"command": "openshift.explorer.manageContext",
+				"title": "Manage Contexts",
```

Comment:
The primary purpose of this feature to switch contexts, so it should be clear what it is and IMO old name should stay.